### ISSUE TITLE
[Documentation] Transaction account details incorrect

### ISSUE THREAD

**Issue by JonasDe** [iron-fish/ironfish#531]
Hi, 

The page:
https://ironfish.network/docs/onboarding/send-receive-iron-fish-transactions

Says:
"E.g. To send 2 coins from the account Savings"
"ironfish accounts:pay -a 1 -o 0.00000001 -t 997c...7fc52ed -f Savings"

But the provided amount (-a 1) is not 2 counts, but 1. So there is a mismatch between description and executed command

**Comment by dguenther** [comment]
Good catch! We have a separate repository for our documentation, so I'm going to move that issue over there.
